scene/3d/reflection_probe.cpp

```diff
@@ -85,38 +85,38 @@ float ReflectionProbe::get_mesh_lod_threshold() const {
 	return mesh_lod_threshold;
 }
 
-void ReflectionProbe::set_extents(const Vector3 &p_extents) {
-	extents = p_extents;
+void ReflectionProbe::set_size(const Vector3 &p_size) {
+	size = p_size;
 
 	for (int i = 0; i < 3; i++) {
-		if (extents[i] < 0.01) {
-			extents[i] = 0.01;
+		if (size[i] < 0.01) {
+			size[i] = 0.01;
 		}
 
-		if (extents[i] - 0.01 < ABS(origin_offset[i])) {
-			origin_offset[i] = SIGN(origin_offset[i]) * (extents[i] - 0.01);
+		if (size[i] - 0.01 < ABS(origin_offset[i])) {
+			origin_offset[i] = SIGN(origin_offset[i]) * (size[i] - 0.01);
```

Comment:
I think this logic has changed. My reading of this is that it is clamping the origin offset to be within the extents. However, with this change origin offset will only be clamped within a range of size from the origin i.e. within 2*size. So this logic needs to be updated to ensure that the behaviour stays the same

_edit: Ah, you made the change already below_
```
if (size[i] / 2 - 0.01 < ABS(origin_offset[i])) {
	origin_offset[i] = SIGN(origin_offset[i]) * size[i] / 2 - 0.01;
```